### PR TITLE
Fix the upper case of DT

### PR DESCRIPTION
**Pull request by Shangxin-Liu**
This fixed the upper case typo of dynamics topography code. @gassmoeller  @ian-r-rose

### CODE REVIEW

**Comment by Shangxin-Liu** [comment]
May need to first get my last pull request of new geoid test prm file merged and then got for this.

File: tests/geoid.prm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # A simple pure math setup to compute geoid in single degree 2 order 0 perturbation density field.
 # The perturbation is added to a constant density, i.e., degree 0 order 0 component.
-# The analytic solution of the denbsity integral contribution of this case should be -5.3059320038
+# The analytic solution of the denbsity integral contribution of this case should be -0.53059320038
```
Comment (bangerth) [inline tests/geoid.prm]: while you're there, might as well fix the typo in "denbsity"

Comment (Shangxin-Liu) [inline tests/geoid.prm]: got it.

**Comment by Shangxin-Liu** [comment]
I resolved the previous conflict and squash to one commit.

**Comment by gassmoeller** [comment]
I agree that this is a good idea, but it is an incompatible change to all parameter files that use this subsection. Therefore, please:

1. Add an entry to modules/changes/ that describes that you changed the subsection name
2. Add the following lines to doc/update_prm_files_to_2.0.0.sed:
```
# Rename Dynamic topography subsection
s/subsection Dynamic Topography/subsection Dynamic topography/g
```
3. Make sure all prm files in the repository are updated to the new names by running the following shell script in the source directory:
```
#!/bin/bash

for file in `find . -name *.prm*`; do
echo $file
sed -i "" -f doc/update_prm_files_to_2.0.0.sed $file
done
```

**Comment by gassmoeller** [comment]
@Shangxin-Liu do you have some time to finish this PR in the next weeks? I think the steps above should already do the trick.

**Comment by Shangxin-Liu** [comment]
It must be long enough to make me forget this:-)  Yes, I'll finish this this week. @gassmoeller

**Comment by Shangxin-Liu** [comment]
When I use your script, I noticed that there is anther change from "Model name" to "List of model names". Why this happen?? @gassmoeller

**Comment by gassmoeller** [comment]
*Sigh*, this happens because linux `sed` and macos `sed` do not speak the same language. I tested this on ubuntu, and did not notice that the commands in lines 27-29 of `doc/update_prm_files_to_2.0.0.sed` are GNU extensions, that are not available for Mac. The simplest solution for you right now is to comment out these lines, revert the changes that the script did, and reapply the script, while we search for a Mac solution.

@tjhei: You have some experience with Mac, do you happen to know how to convert these lines into something apple compatible?

**Comment by Shangxin-Liu** [comment]
I reapply the change. Let's see whether this works. @gassmoeller

**Comment by Shangxin-Liu** [comment]
There seems still some weird "end" refresh in some prm files, why this happens?? @gassmoeller

**Comment by Shangxin-Liu** [comment]
Tests happy now~

**Comment by gassmoeller** [comment]
Great, thanks a lot. :+1: 

Usually, files should end with a line break character. However, some text editors do not include one at the end. There are lengthy arguments about it (see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), but for this case I simply assume `sed` adds a line break at the end of a file, if it finds one without (or it is you `git` version). Whatever it is, it is a correct change.

**Comment by Shangxin-Liu** [comment]
Ok, I got it. I noticed that this also fixed the prm file of onset-of-convection benchmark regarding upper case "Boussinesq approximation" and "Initial temperature model". This also confuses me a little bit. So this benchmark previously didn't get its prm file parameters' name correctly? @gassmoeller

**Comment by gassmoeller** [comment]
It was merged from a pull request that was opened before I added the update file, that is why it 'escaped' the update. There might be others that we merge until the 2.0 release, but I plan on running the script periodically to make sure every file is updated.

**Comment by Shangxin-Liu** [comment]
Ok. This makes me clear. @gassmoeller

**Pull request merged**